.eslintrc.js

```diff
@@ -434,7 +434,7 @@ const base = {
   'sonarjs/no-empty-collection': ERROR,
   // function calls should not pass extra arguments
   'sonarjs/no-extra-arguments': ERROR,
-  // goolean expressions should not be gratuitous
+  // boolean expressions should not be gratuitous
```

Comment:
Some repositories are whimsical; this repository does not appear to be one of them